### DISCUSSION TITLE
Dịch động từ stack

### DISCUSSION THREAD

**Discussion by quangvnai** [mlbvn/handson-ml2-vn#115] [Terminology] [answered]
Trong chương về Ensemble, ta đã để nguyên từ `stacking`.
Ở chương 10, tác giả có đề cập đến động từ `stack` trong 1 phương pháp cho dropout (cũng tương tự ensemble).
Mình thì đang để nguyên là `stack`, anh Thướng thì đang chọn dịch là `gộp`.

Mọi người vote xem nên chọn phương án nào:
1. 👍 stack
2. 🎉 gộp

Context:
> Ta chỉ cần thực hiện 100 phép dự đoán trên tập kiểm tra, đặt \texttt{training=True} để đảm bảo rằng tầng \texttt{Dropout} được kích hoạt, và gộp các dự đoán lại.

**Comment by minhduc0711** [comment, marked as answer]
Mình nghĩ từ `gộp` có thể bị hiểu thành `aggregate`. Từ `stack` ở đây có vẻ chỉ đơn giản là `xếp chồng` các ma trận lên nhau nên mình nghĩ có thể dịch như vậy (hơi khác với `stacking` ở ensemble là nhét output của nhiều mô hình vào 1 mô hình cuối cùng)

**Reply by quangvnai** [reply]
stack vào xong vẫn lấy TB mà? Thế thì cũng có đôi nét ensemble ;))

**Reply by ngcthuong** [reply]
Từ này nên dịch linh động (y)

**Reply by minhduc0711** [reply]
@davidnvq ở đây có bước `np.stack` theo nghĩa đen luôn :v Nói chung là mình nghĩ nên dịch.

**Reply by quangvnai** [reply]
mà `gộp` với `xếp chồng` thì có cần làm rõ 1 tính chất của `np.stack` hay `torch.stack` đó là các tensor phải có cùng kích thước chiều (trừ chiều đang stack) không nhỉ :))

**Reply by minhduc0711** [reply]
Cái đó chắc để độc giả tự đọc docs :v

**Reply by quangvnai** [reply]
vậy chốt từ `xếp chồng` sát nghĩa nhất với từ `stack` ạ :3
